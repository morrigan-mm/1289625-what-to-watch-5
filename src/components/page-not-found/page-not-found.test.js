import React from "react";
import {StaticRouter} from "react-router-dom";
import renderer from "react-test-renderer";
import PageNotFound from "./page-not-found";


describe(`PageNotFound`, () => {
  it(`Should render correctly`, () => {
    const tree = renderer
      .create(
          <StaticRouter>
            <PageNotFound />
          </StaticRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
