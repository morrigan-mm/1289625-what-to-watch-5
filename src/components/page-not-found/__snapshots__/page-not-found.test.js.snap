// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound Should render correctly 1`] = `
<div
  className="user-page"
>
  <header
    className="page-header user-page__head"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
  </header>
  <div
    className="sign-in user-page__content"
  >
    <div
      className="sign-in__message"
    >
      <p>
        Sorry! The page, you are looking for doesn't exist! :(
      </p>
    </div>
  </div>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        Â© 2020 What to watch Ltd.
      </p>
    </div>
  </footer>
</div>
`;
